apex-checks/src/main/resources/org/sonar/l10n/apex/rules/apex/A1024.html

```diff
@@ -4,12 +4,11 @@
   -->
```

Comment:
right, I did create a new html and my check has the code A1025 now, but I accidentally deleted the wrong message. Will fix it right away